quicfire_tools/inputs.py

```diff
@@ -9,10 +9,17 @@
 import importlib.resources
 from pathlib import Path
 from string import Template
-from typing import Literal, Union
+from typing import Literal, Union, List, Optional
 
 # External Imports
 import numpy as np
+import pandas as pd
+import bisect
```

Comment:
Minor thing, but bisect should be a core import

Reply:
What is the difference between core and external?